guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
         ├── Foo.jsx             React component
         ├── Foo.test.jsx        React component tests
         └── Foo.example.jsx     React component used to render an example
+        └── Foo.example.html    HTML example
 ```
 
-# CSS components
+## CSS components
 
 [KSS](https://github.com/kss-node/kss-node) is the primary way documentation is written. They establish what pages get created and the content on those pages. We use KSS to parse CSS comment blocks to extract documentation, and it expects your comment block to follow this particular format:
 
-```css
+```scss
 /*
 [Heading]
 
 [Description]
 
-[@flags]
-
-Markup:
-[Inline HTML/EJS or filename]
+[Flags]
 
 [Modifiers]
 
+Markup: [filename]
+
 Style guide: [Reference]
+
+[Documentation and guidance]
```

Comment:
is KSS looking for this?

Reply:
When documenting a component there are 2 pieces that are needed one is `Style guide: components.alert.guidance`

The other is the actual guidance content which is what the `[Documentation and guidance]` is supposed to represent here.

Reply:
Ok. Maybe add a comment to that effect inline -- it's pretty unclear